website/src/components/Features/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import classnames from 'classnames';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import styles from './styles.module.css';
+
+const features = [
+  {
+    title: <>Define your model</>,
+    imageUrl: 'img/input.png',
+    description: (
+      <>
+        {/* Docusaurus was designed from the ground up to be easily installed and
+        used to get your website up and running quickly. */}
+      </>
+    ),
+  },
+  {
+    title: <>Choose your stack</>,
+    imageUrl: 'img/config.png',
+    description: (
+      <>
+        {/* Docusaurus lets you focus on your docs, and we&apos;ll do the chores. Go
+        ahead and move your docs into the <code>docs</code> directory. */}
+      </>
+    ),
+  },
+  {
+    title: <>Deploy</>,
+    imageUrl: 'img/rocket.png',
```

Comment:
Would it be possible to make it little bit bigger. Taking 8-% of the available space in rectc

Reply:
Still working on that

Reply:
No need to release website actually we can still merge it